packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -270,6 +270,7 @@ const defaultL1Network: L1Network = {
   chainId: 1337
 }
 
+console.log('XXXXXXXX', process.env.NEXT_PUBLIC_IS_E2E_TEST)
```

Comment:
leftover console.log

Reply:
Yeah I was testing it while you were reviewing. Removed now.